src/styles/_header.scss

```diff
@@ -14,6 +14,8 @@
 }
 
 .nav {
+  transition-duration: 300ms;
```

Comment:
add properties to which u set transition

Reply:
Yes, I`ve missed this point.

`s corrected:

&__link:hover {
    transition-duration: 300ms;
    color: #00acdc;
  }